src/pages/payments/index/Payments.tsx

```diff
@@ -76,6 +80,50 @@ export function Payments() {
       label: t('transaction_reference'),
     },
   ];
+  const actions = [
+    (resource: any) => (
+      <DropdownElement
+        onClick={() => {
+          navigate(`${resource.id}/apply`);
+        }}
+      >
+        {t('apply_payment')}
+      </DropdownElement>
+    ),
+    (resource: any) => (
+      <DropdownElement
+        onClick={() => {
+          navigate(`${resource.id}/refund`);
+        }}
+      >
+        {t('refund_payment')}
+      </DropdownElement>
+    ),
+    (resource: any) => (
+      <DropdownElement
+        onClick={() => {
+          axios
+            .get(endpoint('/api/v1/payments/:id/email', { id: resource.id }), {
+              headers: defaultHeaders,
+            })
+            .then((data) => console.log(data));
+        }}
+      >
+        {t('email_payment')}
+      </DropdownElement>
+    ),
+  ];
+  const customBulkActions = [
+    (resource: any) => (
+      <DropdownElement
+        onClick={() => {
+          bulk(resource, 'email').then((data) => console.log(data));
```

Comment:
This is wrong. Please redirect users to the email page for the invoice. 

https://github.com/invoiceninja/ui/blob/main/src/common/routes.tsx#L84